libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```
@@ -275,17 +275,27 @@ export class ExecuteBridgeRequest {
   }
 
   private getApiUrl(action: PostActionEnum | GetActionEnum): string {
-    if (action === PostActionEnum.PREVIEW) {
-      return `http://localhost:${process.env.PORT}`;
-    }
-
-    const apiUrl = process.env.API_INTERNAL_ORIGIN || process.env.API_ROOT_URL;
+    const baseUrl = action === PostActionEnum.PREVIEW 
```

ESLint

[error] Replace `·action·===·PostActionEnum.PREVIEW·` with `⏎······action·===·PostActionEnum.PREVIEW`

(prettier/prettier)